### PR TITLE
[ODS-5089] Github Action Extension build & SQL Express on Windows

### PR DESCRIPTION
**Pull request by gmcelhanon**

### CODE REVIEW

File: .github/workflows/build.ps1 [hunk 1]
```diff
@@ -4,13 +4,72 @@
 # See the LICENSE and NOTICES files in the project root for more information.
 
 $ErrorActionPreference = 'Stop'
+
+$PSVersionTable
+
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-& ([scriptblock]::Create((Invoke-WebRequest -UseBasicParsing 'https://dot.net/v1/dotnet-install.ps1'))) -Channel 3.1 -InstallDir 'C:/Program Files/dotnet'
+
+if ($env:GITHUB_ACTIONS) {
```
Comment (bptillman) [inline .github/workflows/build.ps1:12]: If this script itself is just used for the github actions build, then do we need to have an if/else check like this in place?

**Review by bptillman — COMMENTED** [review]
Just the one comment about the if statement. Otherwise this looks good to me.

File: .github/workflows/build.ps1 (hunk 1 above)
Comment (vimayya) [reply .github/workflows/build.ps1:12]: @bptillman I am guessing @tbmeador-dlp added the condition to be able to run this script from local box, in which case we wouldn't be using localdb

Comment (bptillman) [reply .github/workflows/build.ps1:12]: Understood and that makes sense. Also makes me wonder if this tool might be useful to try for scenarios like this. I haven't tried it yet myself, but it sounds promising. https://github.com/nektos/act

**Review by bptillman — APPROVED** [review]

**Pull request merged**